gis/geographiclib/Portfile

```diff
@@ -18,19 +18,15 @@ long_description \
     UPS, MGRS, and geocentric coordinates, for gravity (e.g., EGM2008), geoid, \
     and geomagnetic field (e.g., WMM2010) calculations, and for solving geodesic problems.
 
-homepage            http://geographiclib.sourceforge.net/
-master_sites        sourceforge:project/geographiclib/distrib
-distname            GeographicLib-${version}
+homepage            https://geographiclib.sourceforge.io/
 
-checksums           rmd160  50c5c7d1eed519fc065879e2233887baf3228280 \
-                    sha256  aec0ab52b6b9c9445d9d0a77e3af52257e21d6e74e94d8c2cb8fa6f11815ee2b \
-                    size    2448498
+checksums           rmd160  9442f33f21db1464ce97a13049ad104667ae04e7 \
+                    sha256  15304369d74727b1bfb3f848a264dd2dd5bdd5d0c17ee05c2340a0244a9295d8 \
+                    size    1450621
 
 depends_build-append    path:bin/doxygen:doxygen
 
 cmake.out_of_source yes
```

Comment:
please update to the latest version of the cmake PG, which is 1.1. 
Amongst other possible improvements, the "out_of_source yes" is now the default.

Reply:
done